README.md

```diff
@@ -115,7 +116,7 @@ jobs:
       - name: Set up Ruby 2.7
         uses: ruby/setup-ruby@v1
         with:
-          ruby-version: 2.7.1
+          ruby-version: 2.7.3
```

Comment:
`ruby/setup-ruby` reads your `.ruby-version` file to determine which ruby version to install FYI, plus you can pass [`bundler-cache: true`](https://github.com/ruby/setup-ruby#single-job), you probably know but I'd love to spread the word :D

Reply:
Good note about the cache - if you have a `.ruby-version` will it override this?

Reply:
I don't think it does :/

Reply:
Bummer. This is intended for beginners to actions & Bridgetown so having the version locked doesn't bother me that much since I think it's pretty clear in the docs.